chainer/functions/local_response_normalization.py

```diff
@@ -41,6 +44,15 @@ def __init__(self, n=5, k=2, alpha=1e-4, beta=.75):
         self.alpha = alpha
         self.beta = beta
 
+    def check_type_forward(self, in_types):
+        type_check.expect(in_types.size() == 1)
+        x_type, = in_types
+
+        type_check.expect(
+            x_type.dtype == numpy.float32,
+            x_type.ndim == 4,
```

Comment:
It is enough that `x_type.ndim` is greater than or equal to 3.  It need not be 4.